admin/release.py

```diff
@@ -303,3 +329,176 @@ def publish_docs_main(args, base_path, top_level):
         sys.stderr.write("%s: Can't publish non-tagged version to production."
                          % (base_path.basename(),))
         raise SystemExit(1)
+
+
+class UploadOptions(Options):
+    """
+    Options for uploading packages.
+    """
+    optParameters = [
+        ["target", None, ARCHIVE_BUCKET,
+         "The bucket to upload packages to."],
+        ["build-server", None,
+         b'http://build.clusterhq.com',
+         "The URL of the build-server."],
+    ]
+
+    def parseArgs(self, version):
+        self['version'] = version
```

Comment:
This throws me every time. Is there any reason this isn't consistent with the `publish-docs` options?